### ISSUE TITLE
Add membershipPointsEarned property to ProgramMembership

### ISSUE THREAD

**Issue by vholland** [schemaorg/schemaorg#2085]
Lots of email messages about program memberships describe the points earned. I propose adding a ```membershipPointsEarned``` property to ProgramMembership.

The markup might looking something like:

```
{
  "@context": "http://schema.org/",
  "@type": "ProgramMembership",
  "name": "ACME customer rewards",
  "membershipPointsEarned": {
    "@type": "QuantitativeValue",
    "name": "Starpoints",
    "value": 500
  }
}
```

The expected type of QuantitativeValue is to allow authors to use their custom name for the point system.

**Comment by danbri** [comment]
Just what the Web needs, care to file a PR?

**Comment by danbri** [comment]
is the intent that it reports the total on some account, or that it reports an additional number of points corresponding to some transaction being reported in the same message?

**Comment by vholland** [comment]
The former. A lot of email messages contain reports of the number of points/miles/stars someone has earned.

**Comment by RichardWallis** [comment]
Implemented in [release 3.5 ](https://schema.org/docs/releases.html#v3.5)
